komodo/symlink/create_links.py

```diff
@@ -43,10 +51,10 @@ def create_symlinks(links_dict):
 
     if not os.path.isdir(root_folder):
         raise ValueError(f"{root_folder} is not a directory or does not exist")
-
+    links_dict_links = links_dict["links"]
     with working_dir(root_folder):
         for dst, src in links_dict["links"].items():
-            _create_link(src, dst, links_dict["links"])
+            _create_link(src, dst, links_dict_links)
```

Comment:
why this change?

Reply:
I used the value multiple places in previous attempt, so I had it as a variable instead. Removing the change now.